Topic0EjercicioN2/src/building.java

```diff
@@ -0,0 +1,25 @@
+
+public class building {
```

Comment:
Class names in mixed case with the first letter of each internal word capitalized